web/services/checks.go

```diff
@@ -37,7 +38,7 @@ type ChecksService interface {
 	// Check catalog services
 	GetChecksCatalog() (models.ChecksCatalog, error)
 	GetChecksCatalogByGroup() (models.GroupedCheckList, error)
-	CreateChecksCatalogEntry(check *models.Check) error
+	CreateChecksCatalogEntry(check *models.Check) error // seems to be never used
```

Comment:
We cuold remove it. I don't know if we are going to create individual check metadata entries at some point.
I guess with the comment is good enough for now, to now overload the PR

Reply:
kept with the comment